.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+  // Use IntelliSense to learn about possible Node.js debug attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Should we keep this?

Reply:
It auto-generated comment. I think it is not important

Reply:
Better to keep it